docker/docker-compose-mini.yml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2016, EMC, Inc.
+
+version: '2'
+
+volumes:
+  dhcp-leases:
+    external: false
+
+services:
+
+  mongo: # 27017
+    image: mongo:latest
+    network_mode: "host"
+    privileged: true
```

Comment:
I can't remember why this was a challenge with `docker-compose`. Some of the RackHD containers need to run in host mode. It became much easier to move everything into the host network.

MongoDB should be able to go to the virtual network, but I think RabbitMQ has to stay on the host. Mainly because nodes try to connect to RabbitMQ in order to communicate with RackHD.